contracts/interfaces/IRegistry.sol

```diff
@@ -2,13 +2,58 @@ pragma solidity ^0.4.23;
 
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "openzeppelin-solidity/contracts/token/ERC20/ERC20Basic.sol";
+import "./IPausable.sol";
 
-contract IRegistry is Ownable {
+contract IRegistry is Ownable, IPausable {
+
+    /*
+    Valid Address Keys
+        TR_Address -> getAddress("TR_Address")
+        STR_Address -> getAddress("STR_Address")
+        MR_Address -> getAddress("MR_Address")
+        POLY_Address -> getAddress("POLY_Address")
+        */
```

Comment:
Could we use a more consistent naming scheme for all of these variables and keys.

Variables should be tickerRegistry / moduleRegistry / securityTokenRegistry and keys should be "TickerRegistry" / "ModuleRegistry" / "SecurityTokenRegistry" - please could you change this in all of the contracts - generally prefer to avoid "_" in variable names, and instead use "_" to indicate a parameter which is passed into a function.

Reply:
And the same for POLY_Address - could we use polyToken everywhere.